apps/glusterfs/zone_filter.go

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright (c) 2018 The heketi Authors
```

Comment:
You might want to change the year here.

Reply:
I think the commit is actually from 2018... ;-(